projects/plugins/wpcomsh/custom-colors/js/classic-background-stats.js

```diff
@@ -1,10 +1,9 @@
-/* global jQuery, _ */ ( function ( $, _ ) {
-	var processEvent,
-		sendClientSideStat,
-		statBucket = 'classic-custom-background';
+/* global _ */
+( function ( $, _ ) {
+	const statBucket = 'classic-custom-background';
 
-	sendClientSideStat = function ( eventName ) {
-		var url =
```

Comment:
I think what's here is probably fine, but just so it's clear:

Keep in mind that the eslint rules will call for doing things that may not be supported in old browsers. `const` and `let`, for example, require Chrome 49+ (released in 2016), Safari 11+ (2017), Firefox 44+ (2016), Edge (2015), etc.

But all over the place we already don't support such old browsers. So the main concern would be if one of these scripts gets loaded on the site's frontend and we want even people with very old browsers to not get a JS error.

Or the other thing would be to set up webpack to transpile and bundle the scripts, then we'd just need to have the right browserslist config specifying which browsers we want to support.